roles/matrix_mult/files/matrix_mult_pod.yml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: matrix-mult
+  name: matrix-mult-{{ gpu_node_hostname }}
+  namespace: default
+spec:
+  restartPolicy: Never
+  containers:
+  - image: quay.io/dfeddema/matrixmultgpu:latest
```

Comment:
could you share the ContainerFile of this image?
I'm wondering what's the best way to deal with the image, and avoid as much as possible a manual step to deploy it.

As an example, for GPU-burn, I chose to start from `nvcr.io/nvidia/cuda:11.2.2-devel-ubi8`, fetch the relevant files and compile on the fly (but I need to move it to `openshift-psap` GitHub instead of mine ...)

@ArangoGutierrez any though?

Reply:
Hi @kpouget Here's a link to my github repo that container the dockerfile to build the image 
https://github.com/dfeddema/matmul-gpu